odoo/tests/common.py

```diff
@@ -1636,7 +1642,11 @@ def browser_js(self, url_path, code, ready='', login=None, timeout=60, cookies=N
             timeout = timeout * 1.5
 
         self.start_browser()
-
+        if watch and self.browser.dev_tools_frontend_url:
+            _logger.warning('watch mode is only suitable for local testing')
+            debug_front_end = f'http://127.0.0.1:{self.browser.devtools_port}{self.browser.dev_tools_frontend_url}'
+            self.browser._chrome_without_limit([self.browser.executable, debug_front_end])
+            time.sleep(3)
```

Comment:
Because opening the dev tools may take some time to load, this will prevent missing the first seconds of the test.

Reply:
Have you looked into the `--wait-for-debugger` cli switch?

Reply:
We would prefer to avoid to much changes in the chrome command line. It may be cleaner but It doesn't looks to work as I would hope:
`google-chrome --headless --remote-debugging-port=5678 --disable-gpu https://www.google.com`
trying to access `http://127.0.0.1:5678/json`, page not found.
nothing appears in chrome://inspect either

I guess there may be another way to connect to the debugger but I'm not sure how it is intended to work, and it looks complicated to implement this in the current browser_js.